doc/conf.py

```diff
@@ -0,0 +1,83 @@
+from pkgmt.github import get_repo_and_branch_for_readthedocs
+
+repository_url, repository_branch = get_repo_and_branch_for_readthedocs(
+    repository_url="https://github.com/ploomber/ploomber-engine",
+    default_branch="master",
+)
+
+###############################################################################
+# Auto-generated by `jupyter-book config`
+# If you wish to continue using _config.yml, make edits to that file and
+# re-generate this one.
+###############################################################################
+author = "Ploomber"
+comments_config = {"hypothesis": False, "utterances": False}
+copyright = "2023"
+exclude_patterns = ["**.ipynb_checkpoints", ".DS_Store", "Thumbs.db", "_build"]
+execution_allow_errors = False
+execution_excludepatterns = []
+execution_in_temp = True
+execution_show_tb = True
+execution_timeout = 180
+extensions = [
+    "sphinx_togglebutton",
+    "sphinx_copybutton",
+    "myst_nb",
+    "jupyter_book",
+    "sphinx_thebe",
+    "sphinx_comments",
+    "sphinx_external_toc",
+    "sphinx.ext.intersphinx",
+    "sphinx_design",
+    "sphinx_book_theme",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx_jupyterbook_latex",
+]
+external_toc_exclude_missing = False
+external_toc_path = "_toc.yml"
+html_baseurl = ""
+html_favicon = ""
+html_logo = "square-no-bg-small.png"
+html_sourcelink_suffix = ""
+html_theme = "sphinx_book_theme"
+html_theme_options = {
+    "search_bar_text": "Search this book...",
+    "launch_buttons": {
+        "notebook_interface": "jupyterlab",
+        "binderhub_url": "https://binder.ploomber.io",
+        "jupyterhub_url": "",
+        "thebe": False,
+        "colab_url": "",
+    },
+    "path_to_docs": "doc",
+    "repository_url": "https://github.com/ploomber/ploomber-engine",
+    "repository_branch": "main",
```

Comment:
replace url and branch with the values returned by get_repo_and_branch_for_readthedocs